proglearn/forest.py

```diff
@@ -89,7 +89,8 @@ def add_task(self, X, y, task_id=None, tree_construction_proportion=None,
             transformer_voter_decider_split=[tree_construction_proportion, 1-tree_construction_proportion, 0],
             num_transformers=self.n_estimators,
             transformer_kwargs = {"finite_sample_correction": finite_sample_correction},
```

Comment:
This is a minor thing but can you please get rid of this line inputting the transformer_kwargs?

Reply:
This was my fault. finite_sample_correction is a voter_kwarg, not a transformer_kwarg.